samples/nrf9160/modem_shell/README.rst

```diff
@@ -379,6 +379,18 @@ Building and running
 
 .. include:: /includes/build_and_run_nrf9160.txt
 
+DK buttons
+==========
+
+The buttons have the following functions:
+
+Button 1:
+    * Pressing of the button: raising a kill/abort signal. I.e. long enough press of the button will kill/abort all supported running commands.
+    * Releasing of the button: resetting a kill/abort signal.
```

Comment:
I'm thinking if this bullet should be removed. Does user need to know about this resetting? It's something that (s)he cannot avoid once (s)he presses the button, and the key event that happens is killing of the process.
I may not fully understand the resetting part so I'm not sure.
@tokangas Opinions?

Reply:
Updated.